### ISSUE TITLE
Any way to specify default x and y?

### ISSUE THREAD

**Issue by synic** [mawie81/electron-window-state#26]
The docs do not show it, and I couldn't glean it from reading the code (I'm a JS noob).

Can we also specify the default coordinates?

**Comment by benkeen** [comment]
No, it doesn't look like it. But agreed: it seems reasonable for the app to provide a default x, y in the same manner that it does width and height. Perhaps placed in the center of the screen. 

But you can easily code around it: just check to see if the x, y values returned in `windowStateKeeper(...)` are undefined or not, if they are, set them. For all future updates, that info will be automatically saved in state by the library.

**Comment by M1kep** [comment]
It looks like the library handles this via a config file stored in the App Data folder (https://electronjs.org/docs/all#appgetpathname)

https://github.com/mawie81/electron-window-state/blob/e76fe1e006af5eb496e7f66194f3eaadcf4bfc77/index.js#L16

https://github.com/mawie81/electron-window-state/blob/e76fe1e006af5eb496e7f66194f3eaadcf4bfc77/index.js#L158
